templates/base_index.html

```diff
@@ -180,6 +180,11 @@ <h1>{{ self.title() }}</h1>
           <a href="{{ url_for('edit_%s' % endpoint_suffix, id=resource[pkey]) }}" class="btn btn-primary" role="button">
             {{ utils.icon('pencil') }} Edit
           </a>
+          {% if endpoint_suffix == "resource" %}
+            <a href="{{ url_for('new_permission') }}" class="btn btn-success" role="button">
```

Comment:
It would be nice to have the resource from the _New Permission_ button preselected in the _Add Permission_ form (i.e. as param in `/permissions/new?resource_id=123`).

See `create_form()` in [resources_controller.py#L291-L293](https://github.com/qwc-services/qwc-admin-gui/blob/master/controllers/resources_controller.py#L291-L293) for an analogous example of preselecting `type` in the _Add Resource_ form.